<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClass</class>
 <widget class="QWidget" name="WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Sales Manager</string>
  </property>
  <widget class="QStackedWidget" name="managerPage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>701</width>
     <height>411</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>621</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>品牌车管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>311</width>
       <height>361</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>307</width>
        <height>305</height>
       </rect>
      </property>
      <attribute name="label">
       <string>出售车辆</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>20</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>厂家：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>50</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>品牌：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>80</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>报价：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>金额：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>22</x>
         <y>110</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数量：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sellFactComBox">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>20</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="sellBandComBox">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>50</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellPriceLineEidt">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>80</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="sellSumLineEdit">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>140</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="sellNumSpinBox">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>110</y>
         <width>40</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>207</x>
         <y>80</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>万</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>207</x>
         <y>140</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>万</string>
       </property>
      </widget>
      <widget class="QLabel" name="sellLastNumLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>110</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余数量：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sellOkBtn">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sellCancelBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 2</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QListWidget" name="dailyList">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>291</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="widget.qrc"/>
 </resources>
 <connections/>
</ui>
